var/spack/repos/builtin/packages/bazel/unix_cc_configure-0.21.0.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/tools/cpp/unix_cc_configure.bzl b/tools/cpp/unix_cc_configure.bzl
+index 2ae10361ba..6ed392443d 100644
+--- a/tools/cpp/unix_cc_configure.bzl
++++ b/tools/cpp/unix_cc_configure.bzl
+@@ -145,11 +145,19 @@ def get_escaped_cxx_inc_directories(repository_ctx, cc, lang_flag, additional_fl
+     else:
+         inc_dirs = result.stderr[index1 + 1:index2].strip()
+ 
+-    return [
++    default_inc_directories = [
+         _prepare_include_path(repository_ctx, _cxx_inc_convert(p))
+         for p in inc_dirs.split("\n")
+     ]
+ 
++    env = repository_ctx.os.environ
++    if "SPACK_DEPENDENCIES" in env:
```

Comment:
@rubendibattista what Spack version are you working with? The latest develop version does not use `SPACK_DEPENDENCIES` so this won't have an effect. There is a `SPACK_INCLUDE_DIRS` variable.